### PR TITLE
consistency updates for pod security standards documentation

### PR DESCRIPTION
**Pull request by apeabody**
- Capitalize (OPA) Gatekeeper and Policy names
- Update `Pods` case in Windows HostProcess for consistency with the Privileged Containers control
- Other minor changes for internal consistency

### CODE REVIEW

File: content/en/docs/concepts/security/pod-security-standards.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ fail validation.
 		<tr>
 			<td style="white-space: nowrap">HostProcess</td>
 			<td>
-				<p>Windows pods offer the ability to run <a href="/docs/tasks/configure-pod-container/create-hostprocess-pod">HostProcess containers</a> which enables privileged access to the Windows node. Privileged access to the host is disallowed in the baseline policy. {{< feature-state for_k8s_version="v1.26" state="stable" >}}</p>
+				<p>Windows Pods offer the ability to run <a href="/docs/tasks/configure-pod-container/create-hostprocess-pod">HostProcess containers</a> which enables privileged access to the Windows node. Privileged access to the host is disallowed in the baseline policy. {{< feature-state for_k8s_version="v1.26" state="stable" >}}</p>
```
Comment (Ritikaa96) [inline content/en/docs/concepts/security/pod-security-standards.md:60]: Is there only one change this PR is proposing? PRs with single-word change, unless essential change is done, aren't recommended.
It will be better to add more improvements to this page wherever it is needed.

**Comment by Ritikaa96** [comment]
Here is [contribution guide](https://kubernetes.io/docs/contribute/) and [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#code-style-inline-code) for reference

File: content/en/docs/concepts/security/pod-security-standards.md (hunk 1 above)
Comment (apeabody) [reply content/en/docs/concepts/security/pod-security-standards.md:60]: Thanks @Ritikaa96, added some other changes for consistency

**Comment by apeabody** [comment]
> Hi @apeabody are you able to address the reviews suggested?

Thanks @Ritikaa96 - Just getting back to this after the holidays :)

**Comment by Ritikaa96** [comment]
@apeabody, it seems like the Netlify [build failure](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/659f26282e9e8b0008ca3837) caused by timeout. Can you attempt to trigger the build once more by executing the command: git commit --amend --no-edit && git push --force-with-lease

**Comment by Ritikaa96** [comment]
Also, still waiting for you to address the reviews and make changes

**Comment by apeabody** [comment]
Thanks @Ritikaa96 - Looks like the checks ran after the force push.

File: content/en/docs/concepts/security/pod-security-standards.md [hunk 2]
```diff
@@ -300,7 +300,7 @@ fail validation.
 			<td><strong>Policy</strong></td>
 		</tr>
 		<tr>
-			<td colspan="2"><em>Everything from the baseline profile.</em></td>
+			<td colspan="2"><em>Everything from the baseline policy</em></td>
```
Comment (Ritikaa96) [inline content/en/docs/concepts/security/pod-security-standards.md:303]: I think keeping it as _profile_ is better so that the reader doesn't get confused.

**Comment by Ritikaa96** [comment]
there is inconsistency in this page with profiles and policies in the overview paragraph of pod security standards as well
Would like to get opinion from sig members about this. Is this done knowingly or we can replace `profiles` with `policies` or vice versa?
/sig security

File: content/en/docs/concepts/security/pod-security-standards.md (hunk 2 above)
Comment (apeabody) [reply content/en/docs/concepts/security/pod-security-standards.md:303]: Hi @Ritikaa96 - I also prefer `profile `, however `policy` is (currently) the more common term on the page.  Happy to change all instances once we have a decision. :)

**Comment by sftim** [comment]
@kubernetes/sig-security-pr-reviews please provide a technical review on this one; we would want to be sure this change is right to merge

File: content/en/docs/concepts/security/pod-security-standards.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ This guide outlines the requirements of each policy.
 typically aimed at system- and infrastructure-level workloads managed by privileged, trusted users.
 
 The Privileged policy is defined by an absence of restrictions. Allow-by-default
-mechanisms (such as gatekeeper) may be Privileged by default. In contrast, for a deny-by-default mechanism (such as Pod
+mechanisms (such as Gatekeeper) may be Privileged by default. In contrast, for a deny-by-default mechanism (such as Pod
 Security Policy) the Privileged policy should disable all restrictions.
```
Comment (tallclair) [inline content/en/docs/concepts/security/pod-security-standards.md:34]: It's sort of weird to reference gatekeeper and the now defunct PSP here. Maybe just remove those 2 sentences instead?

File: content/en/docs/concepts/security/pod-security-standards.md (hunk 1 above)
Comment (tallclair) [inline content/en/docs/concepts/security/pod-security-standards.md:60]:
```suggestion
				<p>Windows Pods offer the ability to run <a href="/docs/tasks/configure-pod-container/create-hostprocess-pod">HostProcess containers</a> which enables privileged access to the Windows host machine. Privileged access to the host is disallowed in the baseline policy. {{< feature-state for_k8s_version="v1.26" state="stable" >}}</p>
```

I'm not sure we're consistent about referring to nodes / machines / hosts.

File: content/en/docs/concepts/security/pod-security-standards.md [hunk 4]
```diff
@@ -475,16 +475,15 @@ Kubelets prior to v1.24 don't enforce the pod OS field, and if a cluster has nod
 {{< /note >}}
 
 ### Restricted Pod Security Standard changes
-Another important change, made in Kubernetes v1.25 is that the  _restricted_ Pod security
+Another important change, made in Kubernetes v1.25 is that the  _Restricted_ policy
```
Comment (tallclair) [inline content/en/docs/concepts/security/pod-security-standards.md:478]: If you want to capitalize the policy name, we should do it everywhere. There are a few other references to the restricted policy, and a bunch to the baseline policy that need capitalization.

**Review by tallclair — COMMENTED** [review]
/approve

For technical review

File: content/en/docs/concepts/security/pod-security-standards.md (hunk 3 above)
Comment (sftim) [reply content/en/docs/concepts/security/pod-security-standards.md:34]: I agree; remove them. Maybe:
```markdown
The Privileged policy is defined by an absence of restrictions. If you define a Pod where the Privileged
security policy applies, the Pod you define is able to bypass typical container isolation mechanisms.
For example, you can define a Pod that has access to the node's host network.
```

File: content/en/docs/concepts/security/pod-security-standards.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/concepts/security/pod-security-standards.md:60]: I prefer ”host machine”.

File: content/en/docs/concepts/security/pod-security-standards.md [hunk 5]
```diff
@@ -475,16 +475,15 @@ Kubelets prior to v1.24 don't enforce the pod OS field, and if a cluster has nod
...
 has been updated to use the `pod.spec.os.name` field. Based on the OS name, certain policies that are specific
 to a particular OS can be relaxed for the other OS.
 
-
 #### OS-specific policy controls
 Restrictions on the following controls are only required if `.spec.os.name` is not `windows`:
 - Privilege Escalation
 - Seccomp
-- Linux Capabilities
+- Capabilities
```
Comment (sftim) [inline content/en/docs/concepts/security/pod-security-standards.md:486]: Why this change?

File: content/en/docs/concepts/security/pod-security-standards.md (hunk 3 above)
Comment (apeabody) [reply content/en/docs/concepts/security/pod-security-standards.md]: Thanks - Applied

File: content/en/docs/concepts/security/pod-security-standards.md (hunk 4 above)
Comment (apeabody) [reply content/en/docs/concepts/security/pod-security-standards.md:478]: Capitalized all occurrences of the three policy names

File: content/en/docs/concepts/security/pod-security-standards.md (hunk 5 above)
Comment (apeabody) [reply content/en/docs/concepts/security/pod-security-standards.md:486]: I changed to align with the name of the control (`Capabilities`) in the Baseline table.

**Comment by divya-mohan0209** [comment]
@apeabody Please perform a rebase so that we can ready this for merge.

**Comment by apeabody** [comment]
> @apeabody Please perform a rebase so that we can ready this for merge.

Thanks @divya-mohan0209! Currently it's showing no conflicts with the base branch, and no option to update branch.  Let me know if you need anything else!

**Comment by sftim** [comment]
Ah, what we'd like you to do is to [squash](https://www.k8s.dev/docs/guide/pull-requests/#squashing) commits.

File: content/en/docs/concepts/security/pod-security-standards.md (hunk 5 above)
Comment (sftim) [reply content/en/docs/concepts/security/pod-security-standards.md:503]: Let's emphasize what kind of capability this is. The table has to be a little more terse to help layout.

Comment (apeabody) [reply content/en/docs/concepts/security/pod-security-standards.md]: Sure

File: content/en/docs/concepts/security/pod-security-standards.md [hunk 6]
```diff
@@ -317,12 +317,12 @@ fail validation.
 			<td><strong>Policy</strong></td>
 		</tr>
 		<tr>
-			<td colspan="2"><em>Everything from the baseline profile.</em></td>
+			<td colspan="2"><em>Everything from the Baseline policy</em></td>
```
Comment (sftim) [inline content/en/docs/concepts/security/pod-security-standards.md:320]: I'll accept this,  but isn't Baseline actually a _standard_ now?

File: content/en/docs/concepts/security/pod-security-standards.md [hunk 7]
```diff
@@ -317,12 +317,12 @@ fail validation.
...
 		</tr>
 		<tr>
 			<td style="white-space: nowrap">Volume Types</td>
 			<td>
-				<p>The restricted policy only permits the following volume types.</p>
+				<p>The Restricted policy only permits the following volume types.</p>
```
Comment (sftim) [inline content/en/docs/concepts/security/pod-security-standards.md:325]: policy or standard? Maybe there's a further fix to make.

**Review by sftim — COMMENTED** [review]
Taking https://github.com/kubernetes/website/pull/44311#pullrequestreview-1932938544 as tech LGTM

I notice the term “policy“ is used a lot, but actually these are now _standards_. You can set a policy requiring Pods to meet a particular standard for security.

/lgtm
/approve
